tests/mocks/valid_docker_stack.yaml

```diff
@@ -2,18 +2,19 @@ version: "3"
 
 services:
   app:
-    image: jenkins:latest
+    image: alpine:latest
```

Comment:
I did not understand why these services changed

Reply:
we dont use jenkins ;) it is only minor